xwiki-platform-core/xwiki-platform-search/xwiki-platform-search-solr/xwiki-platform-search-solr-ui/src/main/resources/Main/SolrSearch.xml

```diff
@@ -845,8 +848,12 @@ body.content.preference-underlining-only-inline-links #xwikicontent .search-face
 }
 
 .search-facet-body .itemCount {
-  padding: .1em 0;
+  padding: .2em;
   margin-left: auto;
+  background-color: $theme.highlightColor;
+  border-radius: 7px; /* Same value as @border-radius-base from Flamingo. */
+  font-size: 0.8em;
```

Comment:
Same question for the font-size.

Reply:
Shouldn't we start adding a common mechanism asap instead of hard-coding them more and more?
iirc we agreed to start using CSS variables, can't we introduce one for this?

Reply:
@manuelleduc 

I'd rather introduce them all at once so that we don't duplicate things or have inconsistent ways to provide them.

If that's okay with you, I think it'd make sense to put this PR on hold (as a draft) as long as [XWIKI-22667](https://jira.xwiki.org/browse/XWIKI-22667) is not done. This ticket could be too high priority to wait for this though.

WDYT?